src/sentry/preprod/api/endpoints/organization_preprod_artifact_assemble.py

```diff
@@ -167,7 +185,8 @@ def post(self, request: Request, project: Project) -> Response:
                     {
                         "state": ChunkFileState.ERROR,
                         "detail": "Failed to create preprod artifact row.",
-                    }
+                    },
+                    status=500,
```

Comment:
i changed this to be a 500 otherwise it would return a 200 and then the sentry-cli would fail with a confusing serialization error because it expects the `missingChunks` field for all 200s

Reply:
Makes sense. How did you tigger `create_preprod_artifact` to fail? If it's going to be something real users can encounter we probably need to forward an actionable error from `create_preprod_artifact`.